onnxruntime/contrib_ops/webgpu/bert/flash_attention.wgsl.template

```diff
@@ -147,6 +148,27 @@ $MAIN {
     var qk_2 : vec4<q_element_t>;
     var qk_3 : vec4<q_element_t>;
     var qk_4 : vec4<q_element_t>;
+#if is_nvidia
+    for (var i : u32 = 0u; i < head_size_vec; i++) {
```

Comment:
the goal is the prevent sg_id >= max_k_step from accessing k_tile right ? It looks like the is_qualcomm path below achieves that. Can we try using that path for Nvidia and measure perf ?